test/e2e/tests/gas-estimates.spec.js

```diff
@@ -0,0 +1,171 @@
+const {
+  convertToHexValue,
+  withFixtures,
+  logInWithBalanceValidation,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Gas estimates generated by MetaMask', function () {
+  const baseGanacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  const preLondonGanacheOptions = {
+    ...baseGanacheOptions,
+    hardfork: 'berlin',
+  };
+  const postLondonGanacheOptions = {
+    ...baseGanacheOptions,
+    hardfork: 'london',
+  };
+
+  describe('Send on a network that is EIP-1559 compatible', function () {
```

Comment:
These tests were already passing (both before and after this PR), but I included them here just to have all four code paths covered in the same file.